packages/reactotron-server/src/server/apollo/resolvers/commands.ts

```diff
@@ -0,0 +1,26 @@
+import { Resolver, Query, Subscription, Root, Args, Arg } from "type-graphql"
+
+import { MessageTypes } from "../../messaging"
+import { Command } from "../../schema"
+import { commands } from "../../datastore"
```

Comment:
I can see a risk of an editor mistake being made here, having `Command` and `commands`. Perhaps this could be renamed to be a little more distinct?

Reply:
Yeah, good call. I am thinking I will rename the `datastore` ones most likely

Reply:
good call for sure!